app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,5 @@
+module Response
+  def json_response(object, status = :ok)
+    render json: object, status: status
+  end
```

Comment:
What does the `json_response` wrapper gain us?

Reply:
not a lot. i had that in there with a few other things, but pared it down...it can be removed